types/datadog-statsd-metrics-collector/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for datadog-statsd-metrics-collector 1.1
+// Project: https://github.com/xzyfer/datadog-statsd-metrics-collector#readme
+// Definitions by: Michael Mifsud <https://github.com/xzyfer>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import dogstatsd = require('node-dogstatsd');
+
+export default class Collector implements dogstatsd.StatsDClient {
```

Comment:
datadog-statsd-metrics-collector uses `module.exports =` not `export default`, so the types should use the Typescript syntax for this: `export = Collector`. And imports should use `import Collector = require('datadog-statsd-metrics-collector');`

Reply:
Thanks. Feedback addressed.

Reply:
Making this changes results in an error that I don't know how to resolve.

```
ERROR: 2:8  expect  TypeScript@next compile error: 
Module '"/home/travis/build/DefinitelyTyped/DefinitelyTyped/types/datadog-statsd-metrics-collector/index"' has no default export.
```

Reply:
You also have to change the import to the above syntax.

Reply:
Thank you. This is now green.